config/configmaps/install-log-regexes-configmap.yaml

```diff
@@ -82,6 +82,11 @@ data:
       - "platform\\.aws\\.userTags.*: Invalid value:.*value contains invalid characters"
       installFailingReason: InvalidAWSTags
       installFailingMessage: You have specified an invalid AWS tag value. Verify that your tags meet AWS requirements and try again.
+    - name: ErrorDeletingIAMRole
+      searchRegexStrings:
+        - "Error deleting IAM Role \\([a-zA-Z0-9+=,.@-]+\\): DeleteConflict: Cannot delete entity, must detach all policies first."
```

Comment:
```suggestion
        - "Error deleting IAM Role .* DeleteConflict: Cannot delete entity, must detach all policies first."
```

Reply:
when we make these too expressive, they become brittle / prone to breaking in subtle ways. Don't be afraid of `.*`